django-stubs/contrib/admin/checks.pyi

```diff
@@ -16,6 +16,6 @@ class ModelAdminChecks(BaseModelAdminChecks):
 class InlineModelAdminChecks(BaseModelAdminChecks):
     def check(self, inline_obj: BaseModelAdmin, **kwargs: Any) -> Sequence[CheckMessage]: ...  # type: ignore
 
-def must_be(type: Any, option: Any, obj: Any, id: Any): ...
-def must_inherit_from(parent: Any, option: Any, obj: Any, id: Any): ...
-def refer_to_missing_field(field: Any, option: Any, obj: Any, id: Any): ...
+def must_be(type: Any, option: Any, obj: Any, id: Any) -> Sequence[CheckMessage]: ...
```

Comment:
I am pretty sure it is not raw `Sequence`. Is it `list` or `tuple`?

Reply:
It's actually `list`, but all other returns in this file are typed as `Sequence`. I'll change all of them to `List` then.